Sources/Hummingbird/Router/TrieRouter.swift

```diff
@@ -91,3 +91,20 @@ import HummingbirdCore
         }
     }
 }
+
+extension RouterPathTrieBuilder.Node {
+    /// Return list of paths and associated values in trie node
+    /// - Parameter prefix: Prefix for path
+    /// - Returns: Array of path values pairs
+    func values(prefix: RouterPath = "/") -> [(value: Value, path: RouterPath)] {
```

Comment:
I don't think we care about performance here, but for the sake of having mentioned it we should consider a syntax with `into values: inout [(value: Value, path: RouterPath)]`. This allows us to limit (re)allocs of arrays.

Reply:
For the sake of readability and given we don't care about performance here I'll keep it as is